scripts/monitoring/auto_scale_workers.py

```
@@ -15,6 +15,25 @@
 
 evalai_endpoint = os.environ.get("API_HOST_URL")
 auth_token = os.environ.get("AUTH_TOKEN")
+authorization_header = {"Authorization": "Bearer {}".format(auth_token)}
+
+
+def start_worker(challenge_id):
+    start_worker_endpoint = "{}/api/challenges/{}/manage_worker/start/".format(
+        evalai_endpoint, challenge_id
+    )
+    response = requests.put(
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)